opba-protocols/xs2a-protocol/src/test/java/de/adorsys/opba/protocol/xs2a/service/xs2a/dto/consent/authenticate/embedded/ProvidePsuPasswordBodyFromCtxTest.java

```diff
@@ -0,0 +1,35 @@
+package de.adorsys.opba.protocol.xs2a.service.xs2a.dto.consent.authenticate.embedded;
+
+import de.adorsys.opba.protocol.xs2a.config.MapperTestConfig;
+import de.adorsys.opba.protocol.xs2a.service.xs2a.context.Xs2aContext;
+import de.adorsys.opba.protocol.xs2a.util.UtilService;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(classes = MapperTestConfig.class)
+public class ProvidePsuPasswordBodyFromCtxTest {
+    public static final String PATH_PREFIX = "mapper-test-fixtures/provide_psu_password_body_from_";
+
+    @Autowired
+    private ProvidePsuPasswordBody.FromCtx mapper;
```

Comment:
Suggest to add a blank line between annotated fields to improve readablity, here and everywhere

Reply:
blank lined between annotated fields were added in all mappers